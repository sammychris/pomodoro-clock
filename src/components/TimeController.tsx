import React from "react";
import ButtonsController from "./ButtonsController";

const TimeController = ({
  breakValue,
  sessionValue,
  incrementBreak,
  decrementBreak,
  incrementSession,
  decrementSession,
}) => {
  return (
    <div id="buttons">
      <ButtonsController
        id="break"
        title="Break Length"
        control={breakValue}
        increment={incrementBreak}
        decrement={decrementBreak}
      />
      <ButtonsController
        id="session"
        title="Session Length"
        control={sessionValue}
        increment={incrementSession}
        decrement={decrementSession}
      />
    </div>
  );
};

export default TimeController;
