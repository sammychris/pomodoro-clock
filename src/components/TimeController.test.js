import { shallow } from "enzyme";
import TimeController from "./TimeController";

const wrapper = shallow(<TimeController />);

it("expect to render timer-controller component", () => {
  expect(wrapper.length).toEqual(1);
});

it("expect to take snap-shot of time-controller component", () => {
  expect(wrapper).toMatchSnapshot();
});
