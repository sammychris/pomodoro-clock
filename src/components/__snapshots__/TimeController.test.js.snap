// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to take snap-shot of time-controller component 1`] = `ShallowWrapper {}`;
